torch/distributed/_spmd/distribute.py

```diff
@@ -388,8 +388,13 @@ def _rebuild_graph(
                     if all(
                         [
                             isinstance(n.target, torch._ops.OpOverload)
-                            and n.target._schema.name.startswith(
-                                "aten::_foreach"
+                            and (
+                                n.target._schema.name.startswith(
+                                    "aten::_foreach"
+                                )
+                                or n.target._schema.name.startswith(
+                                    "aten::_fused_adam"
+                                )
```

Comment:
nit: startswith() accepts a tuple of string.

Reply:
Thanks! Let me add this change on top.